Dockerfile

```diff
@@ -1,32 +1,19 @@
-FROM michaelwetter/ubuntu-1804_jmodelica_trunk
+FROM boptest
```

Comment:
Where is this image? I can't find it in Docker Hub.

Reply:
Ok, I see now, it's built here: https://github.com/ibpsa/project1-boptest/blob/321c29817e8fb19717101d8df018a47012f89718/docker/makefile.

Reply:
Just pointing out that I updated the README to include building the "Base" BOPTEST image.